omrmakefiles/rules.linux.mk

```diff
@@ -32,7 +32,7 @@ endif
 
 ## Header File Dependencies
 ifeq (gcc,$(OMR_TOOLCHAIN))
-  ifeq (x86,$(OMR_HOST_ARCH))
+  ifneq (,$(filter x86 riscv,$(OMR_HOST_ARCH)))
```

Comment:
The list of architectures should be sorted like they are below at line 167.

Reply:
Updated as suggested.